pages/workspace/add-events-from-inspector.mdx

```diff
@@ -5,33 +5,75 @@ title: 'Add Events and properties from Inspector'
 
 import Link from '../../components/Link';
 
-## Inspector Dashboard
+The Inspector compares the events and properties coming in to the tracking plan and detects if there are any of the incoming missing from the tracking plan. Event and properties can both be added to the tracking plan from the <Link passHref href="/workspace/inspector/inspector-issues-view"><a>Issues view</a></Link> and the <Link passHref href="/workspace/inspector/inspector-events-view"><a>Events view</a></Link>.
+
+## Event not in Tracking Plan
+
+### Issues view
+
+When an event is missing from the tracking plan, the issue details of the issue view will offer you to add it to the tracking plan right from there. Click the "Add to Tracking Plan" button and the event and all it's properties are created.
```

Comment:
```suggestion
When an event is missing from the tracking plan the issue details view will offer you to quickly add it to the tracking plan. Click the "Add to Tracking Plan" button and the event and all it's properties are created.
```
Although this is not true today, is it? We're not displaying issuse for unknown events?

Reply:
We're not displayin the issue for missing events, but you can still add the event from the issue details if it is not in the tracking plan.